packages/account/src/Sections/Security/DerivPassword/change-password-form.jsx

```diff
@@ -133,7 +133,12 @@ class ChangePasswordForm extends React.Component {
                                 </FormBody>
                             )}
                             <FormFooter>
-                                {status && status.msg && <FormSubmitErrorMessage message={status.msg} />}
+                                {status && status.msg && (
```

Comment:
```suggestion
                                { status?.msg && (
```

Reply:
Alright, actually this is not my code but I will update it 👍